source/docs/advanced/shipping/colissimo.md

```diff
@@ -5,19 +5,19 @@ title: Colissimo integration
 
 This page contains information about the different ways you can accept shippings with [Colissimo](https://www.laposte.fr/colissimo) in your Front-Commerce application.
 
-## Magento2 module
+## Prerequisites
+
+No matter the backend solution, the Colissimo Front-Commerce modules requires a Map component to display pickup points. Thus, before proceeding, please make sure that you've chosen a Map implementation from [Display a map](/docs/advanced/features/display-a-map.html).
```

Comment:
```suggestion
No matter the backend solution, the Colissimo Front-Commerce modules requires a Map component to display pickup points. Thus, before proceeding, please make sure that you have chosen a Map implementation from [Display a map](/docs/advanced/features/display-a-map.html).
```

Reply:
`you've` seems grammatically correct for `you have` in this case?